install.md

```diff
@@ -319,14 +320,46 @@ apt-get update -qq && apt-get install -y \
   make
 ```
 
+##### Ubuntu 24.04 or higher
+
+Install go from snap package:
+
+```shell
+snap install go --classic
+```
+
+Install the rest of the packages:
+
+```shell
+apt update -qq && apt install -y \
+  libbtrfs-dev \
+  golang-github-containers-common \
+  git \
+  libassuan-dev \
+  libglib2.0-dev \
+  libc6-dev \
+  libgpgme-dev \
+  libgpg-error-dev \
+  libseccomp-dev \
+  libsystemd-dev \
+  libselinux1-dev \
+  pkg-config \
+  go-md2man \
+  crun \
+  libudev-dev \
+  software-properties-common \
+  gcc \
+  make
+```
+
 **Caveats and Notes:**
 
 If using an older release or a long-term support release, be careful to
 double-check that the version of `runc` is new enough (running `runc --version`
 should produce `spec: 1.0.0` or greater), or else build your own.
 
 Be careful to double-check that the version of golang is new enough, version
-1.12.x or higher is required. If needed, newer golang versions are available at
+1.23.x or higher is required. If needed, newer golang versions are available at
```

Comment:
Let's link directly to the value of go.mod here. Otherwise it will get outdated again in the future.

Reply:
I have removed this line and asking to look inside go.mod